community/kernel/src/main/java/org/neo4j/graphdb/Result.java

```diff
@@ -227,15 +227,17 @@
     /**
      * This is the visitor interface you need to implement to use the {@link Result#accept(ResultVisitor)} method.
      */
-    interface ResultVisitor
+    interface ResultVisitor<VisitationException extends Exception>
```

Comment:
Could we have something like `interface ResultsVisitor<E> extends Visitor<Result.ResultRow, E>`?